app/assets/stylesheets/application.scss

```
@@ -38,14 +38,32 @@ div.content-card {
 }
 
 .text-title{
-  margin-top: 30px;
+  margin-top: 50px;
 }
 
 /* フラッシュ */
 .alert-notice {
   @extend .alert-success;
+  padding-top: 70px;
 }
 
 .alert-alert {
   @extend .alert-danger;
 }
+/* navbar */
+.navbar-expand-lg {
+padding-left: 350px;
+
+}
+ .collapse {
+   padding-left: 100px;
+
+ }
+ .navbar {
+   font-size: 13px;
+    padding-bottom: 10px;
+ }
+ .fixed-top {
+
+ }
+ body { padding-top: 50px; }
```

ナビバーの縦幅に合わせましょう（デバッグツールで確認できます）

```suggestion
body {
    padding-top: 56px;
}
```